base/Dockerfile

```diff
@@ -1,10 +1,12 @@
 ARG STACK_TAG=d_latest
 FROM lsstsqre/centos:${STACK_TAG}
 ENV PYTHONUNBUFFERED True
+ARG kafka=python-confluent-kafka
```

Comment:
I think it's a little unusual to use lowercase for an argument name, but it's not wrong.

Reply:
Changed it to upper case.